File: src/test/java/jp/skypencil/spotbugs/cli/AppTest.java
/* Copyright (C) 2020 Kengo TODA */
package jp.skypencil.spotbugs.cli;

import static org.junit.jupiter.api.Assertions.*;

import org.junit.jupiter.api.Test;

class AppTest {
  @Test
  void appHasAGreeting() {
    App classUnderTest = new App();
    assertNotNull(classUnderTest.getGreeting(), "app should have a greeting");
  }
}

File: src/test/kotlin/jp/skypencil/spotbugs/cli/AppTest.kt
/* Copyright (C) 2021 Kengo TODA */
package jp.skypencil.spotbugs.cli

import kotlin.test.Test

class AppTest {
  @Test
  fun printHelp() {
    App.Companion.main(arrayOf("--help"))
  }

  @Test
  fun printVersion() {
    App.Companion.main(arrayOf("--version"))
  }
}
